sklearn/utils/tests/test_class_weight.py

```diff
@@ -226,3 +226,15 @@ def test_compute_sample_weight_errors():
 
     # Incorrect length list for multi-output
     assert_raises(ValueError, compute_sample_weight, [{1: 2, 2: 1}], y_)
+
+
+def test_compute_class_default():
+    # Test for the case where no weight is given for a present class.
+    # Current behaviour appears to assign the unweighted classes a weight of 1.
+    y = np.asarray([2, 2, 2, 3, 3, 4])
+    classes = np.unique(y)
+
+    cw = compute_class_weight(None, classes, y)
```

Comment:
Is this case not tested? I had intended that you specify a dict but it only includes weights for some of the classes

Reply:
@jnothman thanks for the comment! I didn't find test for case without weight. I also added a couple of tests for partly specified weights. I hope, I understood you correctly :sweat_smile: